cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -181,7 +181,7 @@ def setup_vm(configure_fleecing, appliance, provider):
     vm = appliance.collections.infra_vms.instantiate(
         name=random_vm_name(context="report", max_length=20),
         provider=provider,
-        template_name="env-rhel7-20-percent-full-disk-tpl",
+        template_name="env-rhel7-20-percent-full-disk-pvala-tpl",
```

Comment:
I think this should come from data yaml than hardcodding. We can include this under template session for provider. I'm marking this optional as just modifying but please raise separate PR for this.

Reply:
This template won't be used by any other test case unless you want to provision a VM which has less than 20% disk space, which is why I directly used the template name even though it's weird to look at it. But if you insist, we can move it to data yaml.

Reply:
yup better to take it from yaml... It will help in template maintenance as well.